### ISSUE TITLE
FetchError: invalid json response body

### ISSUE THREAD

**Issue by macklinu** [danger/danger-js#348]
I updated to 2.0.0-alpha.9 at work, and the build is now failing. 2.0.0-alpha.7 succeeded without issues. Here is the output of Danger on CI:

```
+ yarn run danger
OK, everything looks good: Jenkins on GitHub
Error:  { FetchError: invalid json response body at https://github.com/redacted/redacted/pull/64.diff reason: Unexpected token N in JSON at position 0
    at /var/lib/jenkins/jobs/redacted/workspace/node_modules/node-fetch/lib/body.js:48:31
    at process._tickCallback (internal/process/next_tick.js:109:7)
  name: 'FetchError',
  message: 'invalid json response body at https://github.com/redacted/redacted/pull/64.diff reason: Unexpected token N in JSON at position 0',
  type: 'invalid-json' }
error Command failed with exit code 1.
```

Here are the contents of `https://github.com/redacted/redacted/pull/64.diff`:

```diff
diff --git a/package.json b/package.json
index 49a2222..bd51aa7 100644
--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
     "connect-history-api-fallback": "1.3.0",
     "cross-spawn": "4.0.2",
     "css-loader": "0.26.1",
-    "danger": "^1.2.0",
+    "danger": "^2.0.0-alpha.9",
     "danger-plugin-jest": "^1.0.1",
     "danger-plugin-jira-issue": "^1.1.0",
     "danger-plugin-no-test-shortcuts": "^2.0.0",
diff --git a/yarn.lock b/yarn.lock
index a8b98b6..7d6d698 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -1053,11 +1053,10 @@ babel-plugin-transform-strict-mode@^6.24.1:
     babel-runtime "^6.22.0"
     babel-types "^6.24.1"
 
-babel-polyfill@^6.20.0:
-  version "6.23.0"
-  resolved "https://registry.yarnpkg.com/babel-polyfill/-/babel-polyfill-6.23.0.tgz#8364ca62df8eafb830499f699177466c3b03499d"
+babel-polyfill@7.0.0-alpha.19:
+  version "7.0.0-alpha.19"
+  resolved "https://registry.yarnpkg.com/babel-polyfill/-/babel-polyfill-7.0.0-alpha.19.tgz#546aabf80c92f4e9314176a64458a6258e5970de"
   dependencies:
-    babel-runtime "^6.22.0"
     core-js "^2.4.0"
     regenerator-runtime "^0.10.0"
 
@@ -2027,18 +2026,15 @@ danger-plugin-tslint@^2.0.0:
   optionalDependencies:
     serve "^5.1.5"
 
-danger@^1.2.0:
-  version "1.2.0"
-  resolved "https://registry.yarnpkg.com/danger/-/danger-1.2.0.tgz#45012a191af890c8bb3879a2f3bdf0ef406e0cef"
+danger@^2.0.0-alpha.9:
+  version "2.0.0-alpha.9"
+  resolved "https://registry.yarnpkg.com/danger/-/danger-2.0.0-alpha.9.tgz#d7e693286a0221ce3551d10b9c97bd1baa160079"
   dependencies:
-    babel-polyfill "^6.20.0"
+    babel-polyfill "7.0.0-alpha.19"
     chalk "^2.0.0"
     commander "^2.9.0"
-    debug "^2.6.0"
+    debug "^3.0.0"
     github "^9.2.0"
-    jest-config "^20.0.0"
-    jest-environment-node "^20.0.0"
-    jest-runtime "^20.0.0"
     jsome "^2.3.25"
     jsonpointer "^4.0.1"
     lodash.find "^4.6.0"
@@ -2048,7 +2044,9 @@ danger@^1.2.0:
     node-fetch "^1.6.3"
     parse-diff "^0.4.0"
     parse-link-header "^1.0.1"
+    pinpoint "^1.1.0"
     rfc6902 "^1.3.0"
+    vm2 patriksimek/vm2
     voca "^1.2.0"
 
 dargs@5.1.0:
@@ -2085,6 +2083,12 @@ debug@2.6.7:
   dependencies:
     ms "2.0.0"
 
+debug@^3.0.0:
+  version "3.0.0"
+  resolved "https://registry.yarnpkg.com/debug/-/debug-3.0.0.tgz#1d2feae53349047b08b264ec41906ba17a8516e4"
+  dependencies:
+    ms "2.0.0"
+
 decamelize@^1.0.0, decamelize@^1.1.1, decamelize@^1.1.2:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/decamelize/-/decamelize-1.2.0.tgz#f6534d15148269b20352e7bee26f501f9a191290"
@@ -3542,7 +3546,7 @@ jest-cli@^20.0.4:
     worker-farm "^1.3.1"
     yargs "^7.0.2"
 
-jest-config@^20.0.0, jest-config@^20.0.4:
+jest-config@^20.0.4:
   version "20.0.4"
   resolved "https://registry.yarnpkg.com/jest-config/-/jest-config-20.0.4.tgz#e37930ab2217c913605eff13e7bd763ec48faeea"
   dependencies:
@@ -3578,7 +3582,7 @@ jest-environment-jsdom@^20.0.3:
     jest-util "^20.0.3"
     jsdom "^9.12.0"
 
-jest-environment-node@^20.0.0, jest-environment-node@^20.0.3:
+jest-environment-node@^20.0.3:
   version "20.0.3"
   resolved "https://registry.yarnpkg.com/jest-environment-node/-/jest-environment-node-20.0.3.tgz#d488bc4612af2c246e986e8ae7671a099163d403"
   dependencies:
@@ -3656,7 +3660,7 @@ jest-resolve@^20.0.4:
     is-builtin-module "^1.0.0"
     resolve "^1.3.2"
 
-jest-runtime@^20.0.0, jest-runtime@^20.0.4:
+jest-runtime@^20.0.4:
   version "20.0.4"
   resolved "https://registry.yarnpkg.com/jest-runtime/-/jest-runtime-20.0.4.tgz#a2c802219c4203f754df1404e490186169d124d8"
   dependencies:
@@ -4863,6 +4867,10 @@ pinkie@^2.0.0:
   version "2.0.4"
   resolved "https://registry.yarnpkg.com/pinkie/-/pinkie-2.0.4.tgz#72556b80cfa0d48a974e80e77248e80ed4f7f870"
 
+pinpoint@^1.1.0:
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/pinpoint/-/pinpoint-1.1.0.tgz#0cf7757a6977f1bf7f6a32207b709e377388e874"
+
 pkg-dir@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/pkg-dir/-/pkg-dir-1.0.0.tgz#7a4b508a8d5bb2d629d447056ff4e9c9314cf3d4"
@@ -6805,6 +6813,10 @@ vm-browserify@0.0.4:
   dependencies:
     indexof "0.0.1"
 
+"vm2@github:patriksimek/vm2":
+  version "3.4.6"
+  resolved "https://codeload.github.com/patriksimek/vm2/tar.gz/0711926ba6e6fbfb89277d8033cfd11b69888e46"
+
 voca@^1.2.0:
   version "1.3.0"
   resolved "https://registry.yarnpkg.com/voca/-/voca-1.3.0.tgz#02751ac839bf0c92e2cfe88e49c393c94dd50ac3"
```

Let me know if you need more information. 👍

**Comment by orta** [comment]
This feels weird - it's JSON parsing the diff, from the looks of that errror

**Comment by orta** [comment]
Fixed with #349 but not released yet
